output-alexa/src/models/card/Card.ts

```diff
@@ -12,6 +21,22 @@ export enum CardType {
 
 export type CardTypeLike = EnumLike<CardType>;
 
+export enum PermissionScope {
+  ReadProfileName = 'alexa::profile:name:read',
+  ReadProfileGivenName = 'alexa::profile:given_name:read',
+  ReadProfileEmail = 'alexa::profile:email:read',
+  ReadProfileMobileNumber = 'alexa::profile:mobile_number:read',
+  ReadWriteReminders = 'alexa::alerts:reminders:skill:readwrite',
+  ReadWriteTimers = 'alexa::alerts:timers:skill:readwrite',
+  ReadList = 'read::alexa:household:list',
+  WriteList = 'write::alexa:household:list',
+  ReadAddressAll = 'read::alexa:device:all:address',
```

Comment:
What about `ReadAddressFull`?

Reply:
https://developer.amazon.com/en-US/docs/alexa/custom-skills/device-address-api.html#getAddress

"Gets the full address associated with the device specified by deviceId."

I think `ReadAddressFull` is reasonable.